apps/relgen/src/index.ts

```diff
@@ -376,11 +376,11 @@
     let prompt: string | undefined;
 
     if (templateFile) {
-      template = await readFile(templateFile, 'utf-8');
+      template = (await readFile(templateFile, 'utf-8')).trim();
     }
 
     if (promptFile) {
-      prompt = await readFile(promptFile, 'utf-8');
```

Comment:
The use of `trim` here should be avoided. Please find an alternative method to handle whitespace.